purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/AcknowledgePurchaseUseCase.kt

```diff
@@ -39,6 +44,21 @@ internal class AcknowledgePurchaseUseCase(
                 processResult(
                     billingResult,
                     useCaseParams.purchaseToken,
+                    onError = {
+                        val underlyingErrorMessage: String
+                        if (it.responseCode == BillingClient.BillingResponseCode.ITEM_NOT_OWNED &&
```

Comment:
Nitpick but maybe add a name to the type in the lambda? Like `billingResult` (Same in the `ConsumePurchaseUseCase`)